src/main/kotlin/no/nav/familie/ef/sak/vedtak/historikk/AndelHistorikkDto.kt

```diff
@@ -40,7 +41,7 @@ data class AndelHistorikkDto(
  * disse skal ikke tas med når man skal plukke ut alle aktive andelene
  */
 fun AndelHistorikkDto.erAktivVedtaksperiode() =
-    this.endring?.type == null || this.endring.type == EndringType.SPLITTET
+    (this.endring?.type == null || this.endring.type == EndringType.SPLITTET) && !erOpphør
```

Comment:
Kunne man snudd på denne for lesbarhet? 

fun AndelHistorikkDto.er**Ikke**AktivVedtaksperiode() = this.endring?.type in listOf(EndringType.FJERNET, EndringType.ERSTATTET) || erOpphør

Vet ikke om det blir bedre...

Reply:
Snudde på denne metoden tidligere fra å være negativ til "erAktivPeriode", for å ikke få `filterNot{ erIkke... }`

Reply:
```suggestion
    !erOpphør && (this.endring?.type == null || this.endring.type == EndringType.SPLITTET)
```

Reply:
Det var alt som skulle til for meg  😄